kolibri/utils/cli.py

```diff
@@ -173,6 +173,16 @@ def initialize(debug=False):
 
     :param: debug: Tells initialization to setup logging etc.
     """
+    if debug:
+        # Set default env to the dev settings instead of base
+        os.environ.setdefault(
+            "DJANGO_SETTINGS_MODULE", DEV_SETTINGS_MODULE
```

Comment:
Not happy about this, because debug is then made impossible without dev dependencies...

Reply:
I mean okay you can do `DJANGO_SETTINGS_MODULE=x kolibri start --debug` but it's a detour :)